src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1305,6 +1343,10 @@ SPECTRE_ALWAYS_INLINE constexpr auto create_from(Box&& box,
  * whose members cannot be modified.  When passed a (mutable) rvalue
  * this function will return a mutable DataBox.
  *
+ * Note that in the rvalue and const lvalue case the output DataBox shares all
+ * items that were not removed with the input DataBox. This means if an item is
+ * mutated in the input DataBox it is also mutated in the output DataBox.
```

Comment:
Isn't this true in every case, not just rvalue and const lvalue?  I guess those are the only "real" cases now (the others are implicitly converted to const lvalue), but this reads as if it is contrasting with other types, particularly when read immediately after the previous paragraph.  Also, this isn't a terribly meaningful statement for the rvalue case, since the input DataBox has been moved out of so mutating items in it is not a meaningful operation.

Reply:
Sorry, I think what I was trying to convey is that in the `const lvalue` case there's aliasing, but this was written before deciding on a name for `create_copy`. I'll clarify :)